userprofile/templates/userprofile/profile.html

```diff
@@ -82,6 +82,16 @@ <h2>{% blocktranslate trimmed %}BRAND_PREFERENCES -- {{ brand_name }}{% endblock
 						class="glyphicon glyphicon-copy"></span><span class="">{% translate "COPY" %}</span></button>
 			</span>
 		</div>
+		<div>
+			<form action="{% url 'regenerate-access-token' %}" method="post" class="form form-floating">{% csrf_token %}
```

Comment:
Does the class `form-floating` do anything here? I couldn't find any styles for it in A+ or Bootstrap code.

Reply:
You're right, I thought it fixed an issue I had with ``form-horizontal``, but it turns out that it isn't needed at all.